### ISSUE TITLE
[TB] Brainstorm on worker module

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#25]
## User story
1. As a SD
2. I want to brainstorm on how to develop this module 
3. So that I can later develop it to follow the virtual line given by Sick Lidar LOC

## Acceptance criteria
* Worker module is defined in more detailed for further sprints

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by MeikeBloecher** [comment]
worker node:
	- subscribes to Mqtt topic
	- subscribes to robot location topic
	- subscribes to line measurement topic
	- function that calculates direction to reach the recieved graph node ( needs position of robot, orientation, destination of graph node)
	- Controller function that drives in direction of calculated function, by checking frequently if still on the line 
		- if not return to line and calculate new direction
		- otherwise continue in direction
	- Function that decodes the special ids along the route and assists the calculator function
	- publishes current location, speed, direction for debugging and further features
